src/commands/cat_maths/quadfunc.py

```diff
@@ -0,0 +1,139 @@
+from discord.ext import commands
+from discord import errors
+import discord
+from discord.ext import commands
+import aiohttp, pathlib
+import numpy as np
+import math
+import matplotlib.pyplot as plt
+
+class quadfunc(commands.Cog, name='quadfunc'):
+    '''
+    Quadratic equation solver
+    made by u/SomebodyNotFamous
+    '''
```

Comment:
The variabe names are still in polish because i dont wanna translate them all :)